frontend/src/Components/AlgorithmPage/AlgorithmPage.js

```diff
@@ -1,27 +1,31 @@
 import React, {Component} from "react";
-import PropTypes from "prop-types";
 
 import Base from "../../Layouts/Base";
 import Content from "../../Layouts/Content";
 
 import AlgInputs from "../AlgInputs/AlgInputs";
+import ProgressBar from "../ProgressBar/ProgressBar";
 import ResultCharts from "../Result/ResultCharts";
-import { dataLoadControll, dataLoadForecast } from "../Api/AlgorithmData";
-import { makeStyles } from "@material-ui/core/styles";
 
-import TextField from "@material-ui/core/TextField";
 
 class AlgorithmPage extends Component {
     constructor(props) {
         super(props);
         this.state = {
             results: [],
+            loading: false
         };
     }
 
+    loadingResults(bool) {
```

Comment:
same argument naming comment as the other.

Reply:
fixed